src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="img"
+      >
+        <img
+          src="images/logo.png"
+          alt="Moyo__Img"
+        />
+      </a>
+
+      <nav>
+        <ul class="ul__section">
+          <li class="nav__section">
+            <a
+              href="#"
+              class="nav__button is-active first"
```

Comment:
Class names should represent the meaning of the content, not the styles or tag names. The class 'first' does not represent the content and should be removed if it's only used for styling purposes.